index.js

```diff
@@ -231,3 +256,23 @@ exports.appActivityAudit = function(isStart, sysname, hostname, username) {
   var audit = new AuditMessage(eIdent, [sysParticipant, userParticipant], null, [sourceIdent]);
   return audit.toXML();
 };
+
+exports.auditLogUsedAudit = function(outcome, sysname, hostname, username, userRole, userRoleCode, auditLogURI, objQuery, objDetails) {
```

Comment:
Afterthought: perhaps you could add a JSDoc comment here. I'm doing that on my branch. It would help to have these functions documented.